packages/settings/index.js

```diff
@@ -0,0 +1,44 @@
+const defaults = {
+	adminUrl: '',
+	countries: [],
+	currency: {
+		code: 'USD',
```

Comment:
Again, I know this wasn't introduced in this PR, but isn't the default currency for core WC `GBP`?